specification/markdownlnt.cfg

```diff
@@ -2,6 +2,7 @@
   "plugins" : {
     "MD033" : { "enabled" : false },
     "MD013" : { "enabled" : false },
+    "MD031" : { "enabled" : false },
```

Comment:
This pr will not pass markdownlinter without this change.  When `spec.md` is created, an `<a>` tag is added to the end of the file causing the linter to fail.

<img width="571" alt="image" src="https://github.com/user-attachments/assets/cdd8258c-7588-4515-8084-75808fb0880b">